code/Dockerfile.gpu

```diff
@@ -4,4 +4,5 @@ RUN apt update && apt install -y python3
 
 COPY check-cuda.py /
 
+CMD ["python3", "check-cuda.py"]
```

Comment:
can you run this on an alpine image?

Reply:
No, there are issues with libs and drivers when running in Alpine.
Cuda is not supported in Alpine